src/logic/calculate.js

```diff
@@ -0,0 +1,10 @@
+import operate from './operate';
+
+const calculate = (data, name) => {
```

Comment:
What happens if `name` equals `"AC"` or `"."` or `"="`?
Kindly cover all the cases.